pkgs/by-name/fa/factorio_achievement_enabler/default.nix

```diff
@@ -0,0 +1,35 @@
+{
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  lib,
+}:
+stdenv.mkDerivation {
+  pname = "FAE_Linux";
+  version = "v0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "UnlegitSenpaii";
+    repo = "FAE_Linux";
+    rev = "v1.3";
+    sha256 = "0wf2raza15bc5dpakqk6cvsc30qn3py21cv99z469zrqnzvfqvwn";
+  };
+
+  nativeBuildInputs = [cmake];
+
+  installPhase = ''
+    runHook preInstall
+    mv out $out
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    description = "Factorio Achievement Enabler for Linux
+
+With this you can get steam achievements while playing the game with mods.";
+    homepage = "https://github.com/UnlegitSenpaii/FAE_Linux/";
+    license = licenses.asl20;
+    maintainers = with maintainers; [henriquelay];
+    platforms = ["x86_64-linux"];
```

Comment:
Is this a necessary restriction? Or would it build on ARM? if so `lib.platforms.linux` would allow both

Reply:
It makes no real sense to support ARM: Factorio is game that does not support ARM and [we won't see support for it anytime soon](https://forums.factorio.com/viewtopic.php?p=624969#p624969). Even if FAE_Linux supports ARM, it would be of limited use since it does not patch an ARM binary.

Reply:
ah, fair enough, hadn't realised it was intel only